components/linear/actions/create-issue/create-issue.mjs

```diff
@@ -11,5 +11,66 @@ export default {
   key: "linear-create-issue",
   description: "Create an issue (OAuth). See the docs [here](https://developers.linear.app/docs/graphql/working-with-the-graphql-api#creating-and-editing-issues)",
   version: "0.3.7",
+  props: {
+    ...createIssue.props,
+    createAsUser: {
+      type: "boolean",
+      label: "Create As User",
+      description: "The user to create the issue as. If not provided, the issue will be created as the authenticated user.",
+      default: true,
+    },
+  },
+  additionalProps: async() => {
+    const { createAsUser } = this;
+    if (createAsUser === false) {
+      return [
+        {
+          username: {
+            type: "string",
+            label: "Username",
+            description: "The username of the user to create the issue as.",
+          },
+          userDisplayIconURL: {
+            type: "string",
+            label: "User Display Icon URL",
+            description: "The display icon URL of the user to create the issue as.",
+            optional: true,
+          },
+        },
+      ];
+    } else {
+      return [];
+    }
+  },
+  async run({ $ }) {
+    const {
+      title,
+      description,
+      teamId,
+      assigneeId,
+      createAsUser,
+      username,
+      userDisplayIconURL,
+    } = this;
+
+    const response =
+      await this.linearApp.createIssue({
+        teamId,
+        title,
+        description,
+        assigneeId,
+        createAsUser: createAsUser === true
+          ? username
+          : undefined,
```

Comment:
The username is only passed if `createAsUser` is **true**, but the prop username is only rendered (via `additionalProps`) if createAsUser is **false**. 

It seems to me that both should require the same boolean value (whether it's true or false, depending on the API requirements), in order to pass the username properly.